apis/v1alpha1/ack-generate-metadata.yaml

```diff
@@ -1,13 +1,13 @@
 ack_generate_info:
-  build_date: "2024-10-21T22:52:48Z"
-  build_hash: ab15f9206796e9660c51695fab0ff07a09ea28e2
+  build_date: "2024-12-04T18:42:10Z"
+  build_hash: 5b95c1667a03835266df2d3f598718dd164184af
   go_version: go1.23.2
-  version: v0.39.1-2-gab15f92
+  version: v0.39.1-8-g5b95c16
```

Comment:
Shouldn't this be runtime 0.40?
/lgtm
/hold

Reply:
this happens in the next PR